corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -1035,7 +1044,7 @@ hqDefine("cloudcare/js/formplayer/menus/views", [
         onBeforeDetach: function () {
             const self = this;
             self.smallScreenListener.stopListening();
-            $(window).off('scroll', self.boundHandleScroll);
+            $(this.scrollContainer()).off('scroll', self.boundHandleScroll);
```

Comment:
I think this would be:
```suggestion
            this.scrollContainer().off('scroll', self.boundHandleScroll);
```
since it's already a jQuery selector.

Reply:
Also realizing this could use `self` instead of `this`. Might be nice to have a small cleanup PR once the urgency is addressed.